src/events/lavalink/rawLavalinkEvent.ts

```diff
@@ -34,14 +38,6 @@ export default class extends Event {
 			return this.client.emit(Events.LavalinkWebsocketClosed, manager, payload);
 		}
 
-		if (isPlayerUpdate(payload)) {
-			return this.client.emit(Events.LavalinkPlayerUpdate, manager, payload);
-		}
-
-		if (isDestroy(payload)) {
-			return this.client.emit(Events.LavalinkDestroy, manager, payload);
-		}
-
 		this.client.emit(Events.Wtf, `${this.kHeader} OP code not recognized: ${payload.op}`);
 		this.client.emit(Events.Error, `           Payload: ${JSON.stringify(payload)}`);
```

Comment:
```suggestion
		this.client.emit(Events.Error, `\t\tPayload: ${JSON.stringify(payload)}`);
```

If we're indenting it

Reply:
Those spaces are there to make it aligned.

![image](https://user-images.githubusercontent.com/24852502/79054576-6d358380-7c46-11ea-8306-fdb2a36b6867.png)